src/containers/tutor-home-page/general-info-step/GeneralInfoStep.jsx

```
@@ -57,6 +61,15 @@
   const { t } = useTranslation()
 
   const handleFieldChange = (fieldName) => (e) => {
+    const { value } = e.target
+
+    handleStepData('generalInfo', {
+      ...contextData,
+      [fieldName]: value
+    })
+    setUnsavedChanges(true)
+    setNeedConfirmation(true)
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)